sql/hive/src/main/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -51,6 +51,7 @@ object TestHive
       "TestSQLContext",
       new SparkConf()
         .set("spark.sql.test", "")
+        .set(SQLConf.CODEGEN_FALLBACK.key, "false")
```

Comment:
Turn it false to ensure it does not hide the actual bugs of our expression codegen that causes compilation falure.